packages/docs/src/components/sidebar/algolia-search/AlgoliaSearch.vue

```diff
@@ -48,7 +49,7 @@ export default class Search extends Vue {
         apiKey: 'be3528055c92da2ea5133b93ed548c6d',
         indexName: 'vuestic',
       }, {
-        inputSelector: '#algolia-search-input',
+        inputSelector: '#algolia-search-input .va-input__content__input',
```

Comment:
![image](https://user-images.githubusercontent.com/23530004/143602290-24c366d1-e34c-4cd3-8347-3aa09d9ed8b2.png)

id should be removed from input-wrapper. This may cause errors in future for end user.

Reply:
fixed it